test/e2e/memcached_test.go

```diff
@@ -7,16 +7,14 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
-	"path/filepath"
 	"testing"
 
 	"github.com/operator-framework/operator-sdk/test/e2e/e2eutil"
+	framework "github.com/operator-framework/operator-sdk/test/e2e/framework"
```

Comment:
new line between 13 and 14.

We usually have three groups of imports. The first is the golang internal ones, the second is the imports from this repo, the third is everything else.